rc/filetype/just.kak

```diff
@@ -38,18 +38,19 @@ define-command -hidden just-indent-on-new-line %{
 # ‾‾‾‾‾‾‾‾‾‾‾‾
 
 add-highlighter shared/justfile regions
+
 add-highlighter shared/justfile/content default-region group
+add-highlighter shared/justfile/content/recipe regex '^@?([\w-]+)((?:\h[\w-]+(?:=(?:"[^\n]*?"))?)*):(?!=)(\h(?:[\w-]+|\([\w-]+\h[\w-]+\)))?' 1:function 2:value 3:type
+add-highlighter shared/justfile/content/globals regex ^([\w-]+)\h*:=\h*(["|'][^\n]*?["|'])((?:\h*\+\h*[\w-]+)*) 1:meta 2:value 3:meta
+add-highlighter shared/justfile/content/operator regex '((^@|\h:=\h|=(?=")|\h\+\h))' 1:operator
```

Comment:
are the spaces around `:=` and `+` really required?

Reply:
Just figured out I can define another `strings` regions within the content regions and then define a `dq` region that matches those strings. Yeah! That will simplify everything a lot...